.github/workflows/presubmit-android.yml

```diff
@@ -0,0 +1,18 @@
+name: Presubmit Android
+
+on: [pull_request]
+
+jobs:
+  build:
+    name: build/android
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Run build script
+        run: |
+          cd build/android && ./build.sh ${TARGET}
```

Comment:
not `-p android` ?

Reply:
It's not the rood build.sh, it's the one from build/android, it does -p for you. It's what we use in Kokoro.

Reply:
Ah got it.